net/bridge.go

```diff
@@ -426,7 +426,24 @@ func (f fastdpImpl) attach(veth *netlink.Veth) error {
 	return odp.AddDatapathInterfaceIfNotExist(f.datapathName, veth.Attrs().Name)
 }
 
-func configureIPTables(config *BridgeConfig, ips ipset.Interface) error {
+// Dummy way to check whether a given ipset exists.
+// TODO(brb) Use "ipset -exist create <..>" for our purpose instead (for some reasons
+// creating an ipset with -exist fails).
+func ipsetExist(ips ipset.Interface, name ipset.Name) (bool, error) {
```

Comment:
This is a handy fn a copy pasted from somewhere else. Where do you think is a good home?

Reply:
I think it can go in `weaveworks/weave/net/ipset/ipset.go`, though there is already a function `Exist()` that does something slightly different.  It should probably be renamed `ExistEntry`.